frontend/packages/kubevirt-plugin/src/actions/provider.ts

```diff
@@ -44,9 +45,11 @@ export const useVmActionsProvider = (vm: VMKind) => {
           start,
           VmActionFactory.Restart(k8sModel, vm, { vmi }),
           pause,
-          migrate,
           VmActionFactory.Clone(k8sModel, vm, { vmi }),
+          migrate,
           VmActionFactory.OpenConsole(k8sModel, vm, { vmi }),
+          // disabled until https://issues.redhat.com/browse/CNV-9746 is implemented
```

Comment:
it's removed until we have better way to do ssh, when we do, we will have a different action